system/include/emscripten/wget.h

```diff
@@ -33,9 +33,13 @@ int emscripten_async_wget2_data(const char* url, const char* requesttype, const
 
 void emscripten_async_wget2_abort(int handle);
 
-// wget "sync"
+// wget "sync".
 
-void emscripten_wget(const char* url, const char* file);
+// Returns 0 on success, non-zero on failure
+int emscripten_wget(const char* url, const char* file);
+// Returns 0 on success, non-zero on failure
+int _emscripten_wget_data(const char* url, void** pbuffer, int* pnum);
```

Comment:
It seems slightly odd to me to add this prefixed version. This won't be used by normal users since the prefix indicates it's for internal use only. But internally we can manage with the older API..?

Reply:
I'm happy to drop this PR in favor of your.